src/snowflake/connector/options.py

```diff
@@ -12,14 +13,15 @@
 installed_pandas = False
 installed_keyring = False
 
+logger = getLogger(__name__)
+
 
 def warn_incompatible_dep(dep_name: str,
                           installed_ver: str,
                           expected_ver: 'pkg_resources.Requirement') -> None:
     warnings.warn(
         "You have an incompatible version of '{}' installed, please install a version that "
-        "adheres to: '{}'".format(dep_name,
-                                  _expected_version),
+        "adheres to: '{}'".format(dep_name, _expected_pyarrow_version),
         stacklevel=2)
```

Comment:
Do we want to use function parameters like `expected_ver` instead of `_expected_pyarrow_version`?

Reply:
Oops, right. My bad!